app/jobs/publish_feed_job.rb

```diff
@@ -19,11 +18,17 @@ def perform(podcast, pub_item)
     # grab the current publishing pipeline.
     return :null if null_publishing_item?(podcast, pub_item)
     return :mismatched if mismatched_publishing_item?(podcast, pub_item)
+
     Rails.logger.info("Starting publishing pipeline via PublishFeedJob", {podcast_id: podcast.id, publishing_queue_item_id: pub_item.id})
 
     PublishingPipelineState.start!(podcast)
-    podcast.feeds.each { |feed| publish_apple(podcast, feed) }
+
+    # Publish each integration for each feed (e.g. apple, megaphone)
+    podcast.feeds.each { |feed| publish_integration(podcast, feed) }
```

Comment:
Any dependencies on ordering here?  (Apple before megaphone, or vice versa?)

Reply:
no, that shouldn't matter, but it does do all integrations before RSS publishing, so if it blocks on RSS that is last